charts/coredns/coredns-pod-delete/engine.yaml

```diff
@@ -29,7 +29,4 @@ spec:
         
             # set chaos interval (in sec) as desired
             - name: CHAOS_INTERVAL
-              value: '10'
-                
-            - name: CHAOS_NAMESPACE
-              value: 'kube-system'
```

Comment:
I think we need this env and kube-system namespace is the default and mandatory namespace

Reply:
Is this namespace is different from the namespace where we will install our chaos-engine for this experiment??

Reply:
so, if we don't pass will it take chaos engine namespace

Reply:
yes